src/lib.rs

```diff
@@ -292,3 +297,39 @@ pub fn log_file<P: AsRef<Path>>(path: P) -> io::Result<File> {
         .append(true)
         .open(path)
 }
+
+/// Convenience method for opening a re-openable log file with common options.
+///
+/// The file opening is equivalent to:
+///
+/// ```no_run
+/// std::fs::OpenOptions::new()
+///     .write(true)
+///     .create(true)
+///     .append(true)
+///     .open("filename")
+/// # ;
+/// ```
+///
+/// See [`OpenOptions`] for more information.
+///
+/// [`OpenOptions`]: https://doc.rust-lang.org/std/fs/struct.OpenOptions.html
+#[cfg(all(not(windows), feature = "reopen-03"))]
+#[inline]
+pub fn log_reopen(path: &PathBuf, signal: Option<libc::c_int>) -> io::Result<reopen::Reopen<File>> {
+    let p = path.to_owned();
+    let r = reopen::Reopen::new(Box::new(move || { 
+        OpenOptions::new()
```

Comment:
If it's possible, calling `log_file()` here would mean slightly less duplicated code. Not a big deal if it isn't, though.